Makefile.core.mk

```diff
@@ -79,7 +79,12 @@ GINKGO_FLAGS := $(if $(VERBOSE),-v) $(if $(CI),--no-color)
 # To re-generate a bundle for other specific channels without changing the standard setup, you can:
 # - use the CHANNELS as arg of the bundle target (e.g make bundle CHANNELS=candidate,fast,stable)
 # - use environment variables to overwrite this value (e.g export CHANNELS="candidate,fast,stable")
-CHANNELS ?= ${MINOR_VERSION}
+CHANNEL_SUFFIX := dev
```

Comment:
Looking at channel naming best [practice](https://olm.operatorframework.io/docs/best-practices/channel-naming/#example-2), this should be probably a prefix.

Reply:
fixed!